build_tools/travis/test_script.sh

```diff
@@ -67,3 +59,17 @@ fi
 if [[ "$SKIP_TESTS" != "true" ]]; then
     run_tests
 fi
+
+if [[ "$CHECK_PYTEST_SOFT_DEPENDENCY" == "true" ]]; then
+    conda remove -y py pytest || echo "pytest not found by conda"
```

Comment:
This error message will look weird without someone realising that we're trying to remove it. Silent is fine, IMO.